pkg/cmd/show.go

```diff
@@ -247,7 +248,7 @@ func showVpnShoot(targetReader TargetReader) {
 func showPrometheus(targetReader TargetReader) {
 	username, password = getMonitoringCredentials()
 	showPod("prometheus", "seed", targetReader)
-	output, err := ExecCmdReturnOutput("bash", "-c", "export KUBECONFIG="+KUBECONFIG+"; kubectl get ingress prometheus -n "+getTechnicalID())
+	output, err := ExecCmdReturnOutput("kubectl", "--kubeconfig="+KUBECONFIG, "get", "ingress", "prometheus", "-n", getTargetInfo["shootTechnicalID"])
```

Comment:
instead reading the value directly from the map, I would call a getter. Within the getter you can add proper error handling in case the value does not exist (return error or exit process if you do not expect this to happen usually)

Reply:
done

Reply:
This is not what I meant. I will give you a concrete example what I was thinking about.

E.g., instead of:
```go
pathToKubeconfig = filepath.Join(pathGardenHome, "cache", gardenName, "seeds", getTargetInfo["seed"], "kubeconfig.yaml")
```

call a getter `getFromTargetInfo`:
```go
pathToKubeconfig = filepath.Join(pathGardenHome, "cache", gardenName, "seeds", getFromTargetInfo("seed"), "kubeconfig.yaml")
```
and implement the getter `getFromTargetInfo` like this:
```go
func getFromTargetInfo(kind string) string {
	value := targetInfo[kind]
	if value == "" {
		fmt.Printf("kind %s not found in targetInfo\n", kind)
		os.Exit(2)
		// Alternatively: return error and handle from caller...
	}
	return value
}
```

the above is an example for what I meant with
> instead reading the value directly from the map, I would call a getter. Within the getter you can add proper error handling in case the value does not exist (return error or exit process if you do not expect this to happen usually)